sdk/src/types/block/rand/output/feature.rs

```diff
@@ -36,6 +38,11 @@ pub fn rand_tag_feature() -> TagFeature {
     TagFeature::new(bytes).unwrap()
 }
 
+/// Generates a random [`BlockIssuerFeature`].
+pub fn rand_block_issuer_feature() -> BlockIssuerFeature {
+    BlockIssuerFeature::new(rand_number::<u64>(), rand_public_keys(1)).unwrap()
```

Comment:
How about this?
```suggestion
    BlockIssuerFeature::new(rand_number::<u64>(), rand_public_keys(rand_number_range(BlockIssuerFeature::KEY_COUNT_MIN..=BlockIssuerFeature::KEY_COUNT_MAX))).unwrap()
```

Reply:
but the first param is an expire slot. It's not related to the number of public keys.

Reply:
My suggestion doesn't change the first value

Reply:
oh, sorry, your're right